src/app/creative-team/dashboard/graphs/graphs.component.html

```diff
@@ -0,0 +1,22 @@
+<div fxLayout="row" fxLayoutAlign="start center" fxLayoutGap="10px">
+  <mat-card *ngFor="let graph of graphs" fxFlex="31.3">
```

Comment:
code format please. It have to be in this sequence  fxFlex="31.3" *ngFor="let graph of graphs"